starboard/shared/stub/string_format_wide.cc

```diff
@@ -14,9 +14,11 @@
 
 #include "starboard/common/string.h"
 
+#if 0
```

Comment:
I assume this should be a SB_API_VERSION check (if not, fully remove this block instead)